src/HexCoding.h

```diff
@@ -94,4 +94,22 @@ inline Data parse_hex(const std::string& string, bool padLeft = false) {
     return parse_hex(string.begin(), string.end());
 }
 
+/// "Normalizes" a hexadecimal string (redundant leading zeros are removed, even length is ensured). 
```

Comment:
- hex(Data) will always return even digits, and the same number of bytes as the input.
- hex(uint64_t) will always return 8 bytes (16 hex digits), possibly with leading zeroes.
- odd-length check is not needed
- I prefer dropping this method, and using alternative, as suggested in the two places where it is used

Reply:
Thanks for the suggestion of using `store()`. I've dropped this method (and its tests).